### ISSUE TITLE
World difficulty and resource quantity settings 

### ISSUE THREAD

**Issue by sunxiaoyi111** [lloesche/valheim-server-docker#636]
there Added modifiers for world difficulty and resources in the latest version of the experience server.
Would be sweet if there was a setting environment variable function to set this config.

this settings was saved in Mywordtest.fwl
e.g:
Mywordtest
HA5vzw8RED^徜j民H          playerdamage 85enemydamage 150enemyspeedsize 110deathkeepequipskillreductionrate 15resourcerate 300eventrate 60Npreset combat_hard:deathpenalty_casual:resources_most:raids_more:raids_default

**Comment by agxs** [comment]
Does the dedicated server even support setting these yet via parameters?

**Comment by UmutAlihan** [comment]
yes I am looking forward to somehow apply the new "world modifier" feature on the worlds generated by dedicated-servers

**Comment by wrender** [comment]
What I did was just generate the world with the "hard" settings on desktop/steam install. Then copy the map over to the dedicated server.  I think this works, but I wasn't sure how to confirm the settings are imbedded in the world file.

**Comment by agxs** [comment]
I did the same and can confirm doing this works.

**Comment by sunxiaoyi111** [comment]
> Does the dedicated server even support setting these yet via parameters?

Not yet in the current Docker version。
As mentioned by  above, currently the world map can only be modified manually by modifying configuration items now。

We really need a convenient configuration method, such as environment variables, to configure map settings

**Comment by N520** [comment]
As of the latest patch, it is possible to configure the server using startup arguments. So something like this
```bash
docker run -d \
    ...
    -e SERVER_ARGS="-crossplay -preset casual" \
    lloesche/valheim-server
``` 
is now possible.  

I attached the updated dedicated server manual for the other, new arguments
[Valheim Dedicated Server Manual.pdf](https://github.com/lloesche/valheim-server-docker/files/12820591/Valheim.Dedicated.Server.Manual.pdf)

**Comment by UmutAlihan** [comment]
that was exactly what I was waiting, such a great vibrant community around this repo, thank you very much!

**Comment by blaklabz** [comment]
this link is exactly what i was hunting for as well!
